src/backend/metal/CommandBufferMTL.mm

```diff
@@ -144,7 +144,7 @@ void BeginSubpass(id<MTLCommandBuffer> commandBuffer, uint32_t subpass) {
                 }
 
                 render = [commandBuffer renderCommandEncoderWithDescriptor:descriptor];
-                [descriptor release];
```

Comment:
As discussed, let's do these changes in a separate CL.

Reply:
Done